src/briefcase/integrations/android_sdk.py

```diff
@@ -314,7 +586,14 @@ def install_apk(self, apk_path):
 
         Returns `None` on success; raises an exception on failure.
         """
-        return self.run("install", str(apk_path))
+        try:
+            self.run("install", str(apk_path))
+        except subprocess.CalledProcessError:
+            raise BriefcaseCommandError(
```

Comment:
Have you seen this happen? If so, great. If not, then maybe we're not catching the error condition accurately.

Reply:
This specific case, no - adding this was part of the refactoring that made `ADB.run` re-raise the `CalledProcessError` for anything other than an invalid device ID. I can't think of any obvious way that this would manifest on install - but as a safety mechanism, it's here.